### ISSUE TITLE
[spring-boot] Mission introduction is not correct

### ISSUE THREAD

**Issue by cmoulliard** [fabric8-launch/appdev-documentation#815]
This text 

```
Missions are working applications that showcase different fundamental pieces of building cloud native applications and services, such as creating HTTP APIs, interoperating with a database, or implementing the health check pattern. A booster is the implementation of a mission in a specific runtime. Boosters are preconfigured, functioning applications based on a mission that demonstrate a fundamental aspect of modern application development running in an environment similar to production. Missions and boosters can be used as a proof of technology demonstration, a teaching tool, or even a sandbox for understanding how to develop applications. They can also be updated or extended for your own use case.
```

should be replaced by

```
Missions are working applications that showcase different fundamental pieces of building cloud native applications (as presented within the concept chapter) and services, such as creating REST, 
interoperating with a database or implementing the health check pattern. The mission can be used as a proof of technology demonstration, a teaching tool, or even a sandbox for understanding how to develop application in order to develop your own project. They can also be updated or extended for your own use case.

NOTE: A booster, by convention, is the implementation of a mission in a specific runtime and by consequent, it will be used within the next sections

```

**Comment by lavernw1** [comment]
updated wording for missions chapter 2

**Comment by tradej** [comment]
Closed via #856.
